moj-node/assets/javascript/audioPlayer.js

```diff
@@ -39,8 +40,12 @@ $(document).ready(function() {
 
   function analyticsAudioEvent(config) {
     return function() {
-      if (!_paq) return;
-      _paq.push(['trackEvent', 'Radio', config.action, config.name]);
+      sendEvent({
+        category: 'Radio',
+        action: config.action,
+        label: config.label,
+        value: 1,
```

Comment:
Does this work in IE11? if not we need to add this to the `.prettierignore`

Reply:
Sorry too much swapping between node and front end 😃

Reply:
Fixed